README.md

```diff
@@ -1,3 +1,92 @@
+# Azure Functions .NET Isolated
```

Comment:
Called it .NET Isolated instead of .NET 5 here. The description below calls out that this provides the .NET 5 support in Functions.